components/post_view/post/post.jsx

```diff
@@ -412,6 +417,50 @@ export default class Post extends React.PureComponent {
             centerClass = 'center';
         }
 
+        const postHeader = (
+            <PostHeader
+                post={post}
+                handleCommentClick={this.handleCommentClick}
+                handleCardClick={this.handleCardClick}
+                handleDropdownOpened={this.handleDropdownOpened}
+                compactDisplay={this.props.compactDisplay}
+                isFirstReply={this.props.isFirstReply}
+                showTimeWithoutHover={!hideProfilePicture}
+                hover={(this.state.hover || this.state.a11yActive || this.state.fileDropdownOpened) && !this.props.isBeingEdited}
+                isLastPost={this.props.isLastPost}
+            />
+        );
+
+        const postBody = (
+            <PostBody
+                post={post}
+                handleCommentClick={this.handleCommentClick}
+                compactDisplay={this.props.compactDisplay}
+                isCommentMention={this.props.isCommentMention}
+                isFirstReply={this.props.isFirstReply}
+                handleFileDropdownOpened={this.handleFileDropdownOpened}
+            />
+        );
+
+        const threadFooter = isCollapsedThreadsEnabled && !post.root_id && (hasReplies || post.is_following) ? <ThreadFooter threadId={post.id}/> : null;
+
+        const showSlot = isBeingEdited ? 2 : 1;
```

Comment:
maybe set this numeric values in constant with meaningful names?

Reply:
done!